include/swift/AST/Builtins.def

```diff
@@ -113,7 +113,7 @@ BUILTIN_CAST_OR_BITCAST_OPERATION(SExtOrBitCast,  "sextOrBitCast",  "n")
 #ifndef BUILTIN_BINARY_OPERATION_ALL
 #define BUILTIN_BINARY_OPERATION_ALL(Id, Name, Attrs, Overload) \
   BUILTIN_BINARY_OPERATION_OVERLOADED_STATIC(Id, BUILTIN_BINARY_OPERATION_GENERIC_HELPER_STR(Name), Attrs, Overload) \
-  BUILTIN_BINARY_OPERATION_POLYMORPHIC(Generic##Id, BUILTIN_BINARY_OPERATION_GENERIC_HELPER_STR(generic_##Name), Attrs)
+  BUILTIN_BINARY_OPERATION_POLYMORPHIC(Generic##Id, BUILTIN_BINARY_OPERATION_GENERIC_HELPER_STR(generic_##Name), "")
```

Comment:
I would rather change BUILTIN_BINARY_OPERATION_POLYMORPHIC to not take that flag at all.

Reply:
Good idea. I'll do that.